spec/src/lib.rs

```diff
@@ -52,6 +52,7 @@ pub struct ChainSpec {
 pub struct Params {
     pub epoch_reward: Capacity,
     pub secondary_epoch_reward: Capacity,
+    #[serde(default)]
```

Comment:
Maybe not related to this PR, I think that we should move `default` to struct `Params` level, testnet/mainnet use the default value including `max_block_size`, other specs ( dev / integration) can use the different configurable values.

Reply:
I think maybe we should just remove `params` fields from `testnet.toml`, since actually we do not use this config. @doitian

Reply:
Apply the change, also remove the `params` from `testnet.toml`.